pkg/util/coverage/coverage_disabled.go

```diff
@@ -0,0 +1,29 @@
+// +build !coverage
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package coverage
+
+// InitCoverage is a no-op when not running with coverage.
+func InitCoverage(name string) {
```

Comment:
should this panic instead?

Reply:
As originally envisioned, it was reasonable for this to be called from places other than the dummy test. Given that this is no longer done, and I cannot see a reason it should happen, I think that's a reasonable change to make.